README.md

```diff
@@ -1,121 +1,103 @@
-Herman [a SassDoc theme]
-========================
+# Herman [a SassDoc theme]
 
 [![CircleCI](https://circleci.com/gh/oddbird/sassdoc-theme-herman.svg?style=svg)](https://circleci.com/gh/oddbird/sassdoc-theme-herman)
 [![Coverage Status](https://coveralls.io/repos/github/oddbird/sassdoc-theme-herman/badge.svg)](https://coveralls.io/github/oddbird/sassdoc-theme-herman)
 [![Greenkeeper badge](https://badges.greenkeeper.io/oddbird/sassdoc-theme-herman.svg)](https://greenkeeper.io/)
```

Comment:
Minor point, but do we want some padding in between these?

![screenshot 2017-11-08 21 41 28](https://user-images.githubusercontent.com/552316/32585914-c04f99bc-c4cd-11e7-9421-f2018e51a364.jpg)

Reply:
Weird, now it looks fine to me after being merged to `master`...

Reply:
Yeah, we could try adding `&nbsp;` between them?